packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -24,54 +25,65 @@ const Notification = ({ data, removeNotificationMessage }) => {
         setTimeout(destroy, data.delay || default_delay);
     }
 
-    return (
-        <div
-            className={classNames('notification', types[data.type], {
-                'notification--small': data.size === 'small',
-            })}
-        >
-            <div className='notification__icon-background'>
-                <NotificationStatusIcons type={data.type} class_suffix='is-background' />
-            </div>
-            <div className='notification__icon'>
-                <NotificationStatusIcons type={data.type} />
-            </div>
-            <div className='notification__text-container'>
-                <h4 className='notification__header'>{data.header}</h4>
-                {data.timeout && (
-                    <LinearProgress
-                        className='notification__timeout'
-                        timeout={data.timeout}
-                        action={data.action.onClick}
-                        render={data.timeoutMessage}
-                    />
-                )}
-                <p className='notification__text-body'>{data.message}</p>
-                <div className='notification__action'>
-                    {!isEmptyObject(data.action) && (
-                        <React.Fragment>
-                            {data.action.route ? (
-                                <BinaryLink
-                                    className={classNames('dc-btn', 'dc-btn--secondary', 'notification__cta-button')}
-                                    to={data.action.route}
-                                >
-                                    <span className='dc-btn__text'>{data.action.text}</span>
-                                </BinaryLink>
-                            ) : (
-                                <Button
-                                    className='notification__cta-button'
-                                    onClick={() => data.action.onClick({ is_deriv_crypto })}
-                                    text={data.action.text}
-                                    secondary
-                                />
+    switch (data.type) {
+        case 'dp2p':
```

Comment:
Is there another way so we can avoid handling platform specific exceptions in `core`? 
Instead, maybe we can add refactor the banner into a shareable component and call it from here as a different style of notification?

Reply:
Understood, will refactor it into a general notification banner for `news` so that any new products can utilise it as well.